### PR TITLE
Fix Nightly Builds

### PR DESCRIPTION
**Pull request by LTurkingtonHMCTS**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by nickwest1 — APPROVED** [review]

**Review by MohammedHMCTS — APPROVED** [review]

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -91,4 +91,18 @@
     <packageUrl regex="true">^pkg:maven/io\.netty/netty.*@4\.1\.69\.Final$</packageUrl>
     <cve>CVE-2022-24823</cve>
   </suppress>
+  <suppress until="2022-07-01">
+    <notes><![CDATA[
+   file name: common-image-3.7.0.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.twelvemonkeys\.common/common\-.*$</packageUrl>
```
Comment (MohammedHMCTS) [inline config/owasp/suppressions.xml:98]: Did you add 2 of the same suppressions?

**Review by MohammedHMCTS — CHANGES_REQUESTED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (TeaWithLucas) [reply config/owasp/suppressions.xml:98]: No, different regexs

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -91,4 +91,18 @@
...
+    <cve>CVE-2021-23792</cve>
+  </suppress>
+  <suppress until="2022-07-01">
+    <notes><![CDATA[
+   file name: common-image-3.7.0.jar
```
Comment (TeaWithLucas) [inline config/owasp/suppressions.xml:103]: @LTurkingtonHMCTS can you update this description to the package it relates to?

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (LTurkingtonHMCTS) [reply config/owasp/suppressions.xml]: different suppressions but streamlined the regex to include both

**Review by MohammedHMCTS — APPROVED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (wmarshid) [reply config/owasp/suppressions.xml]: that's what I call being a boss @LTurkingtonHMCTS 💪

**Review by wmarshid — APPROVED** [review]

**Pull request merged**